recipes-extended/xen/files/libxl-openxt-qemu-args.patch

```diff
@@ -90,15 +90,7 @@ PATCHES
          if (libxl_defbool_val(b_info->u.hvm.gfx_passthru)) {
              switch (b_info->u.hvm.gfx_passthru_kind) {
              case LIBXL_GFX_PASSTHRU_KIND_DEFAULT:
-@@ -665,6 +657,7 @@ static const char *qemu_disk_format_stri
-     case LIBXL_DISK_FORMAT_QED: return "qed";
-     case LIBXL_DISK_FORMAT_HOST_CDROM: return "host_cdrom";
-     case LIBXL_DISK_FORMAT_HOST_DEVICE: return "host_device";
-+    case LIBXL_DISK_FORMAT_FILE: return "file";
```

Comment:
Not needed?

Reply:
No, the function was moved to a different file which I did not refresh properly it seems. Good catch.